dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -1001,18 +1000,10 @@ public SourceFile getDescriptorFromServer(String entry, String descriptorType) t
         }
 
         if (container != null) {
-            try {
-                if (descriptorType.equals(CWL_STRING)) {
-                    file = containersApi.cwl(container.getId(), tag);
-                } else if (descriptorType.equals(WDL_STRING)) {
-                    file = containersApi.wdl(container.getId(), tag);
-                }
-            } catch (ApiException ex) {
-                if (ex.getCode() == HttpStatus.SC_BAD_REQUEST) {
```

Comment:
It's not entirely clear to me, after removing the exception handling, what is the terminal output on the command-line that a user would see?

Reply:
Before it said `Invalid tag`, then `Invalid version` with the stack trace. Now there is only one message with a stack trace, and the message includes the tag name:

Example:

```
dockstore tool cwl --entry quay.io/coverbeckucsc/docstore-tool-md5sum:1.0.5
io.swagger.client.ApiException: Invalid or missing tag 1.0.5.
	at io.swagger.client.ApiClient.invokeAPI(ApiClient.java:713)
	at io.swagger.client.api.ContainersApi.cwl(ContainersApi.java:207)
	at io.dockstore.client.cli.nested.ToolClient.getDescriptorFromServer(ToolClient.java:1006)
	at io.dockstore.client.cli.nested.AbstractEntryClient.handleDescriptor(AbstractEntryClient.java:313)
	at io.dockstore.client.cli.nested.AbstractEntryClient.descriptor(AbstractEntryClient.java:436)
	at io.dockstore.client.cli.nested.AbstractEntryClient.processEntryCommands(AbstractEntryClient.java:237)
	at io.dockstore.client.cli.Client.run(Client.java:689)
	at io.dockstore.client.cli.Client.main(Client.java:602)
```
io.swagger.client.apiexception: Invalid or missing tag 1.0.5.at io.swagger.client.ApiClient.invokeAPI(ApiClient.java:713)at io.swagger.client.api.ContainersApi.cwl(ContainersApi.java:213)at io.dockstore.client.cli.nested.ToolClient.getDescriptorFromServer(ToolClient.java:1004)at io.dockstore.client.cli.nested.AbstractEntryClient.handleDescriptor(AbstractEntryClient.java:313)at io.dockstore.client.cli.nested.AbstractEntryClient.descriptor(AbstractEntryClient.java:436)at io.dockstore.client.cli.nested.AbstractEntryClient.processEntryCommands(AbstractEntryClient.java:237)at io.dockstore.client.cli.Client.run(Client.java:691)at io.dockstore.client.cli.Client.main(Client.java:604)```

Reply:
ok, cool